### DISCUSSION TITLE
Issue about enable merge

### DISCUSSION THREAD

**Discussion by ynguyen2k2** [cungminh2710/coderum.dev#76] [Q&A]
Hi everyone, I meet problem about github action build. Could you explain to help me. Thanks for answering
![image](https://github.com/cungminh2710/coderum.dev/assets/73071640/606672e2-81f9-4c5b-944b-f7066721676e)
![image](https://github.com/cungminh2710/coderum.dev/assets/73071640/62392e43-9683-4a24-ad04-bf40b01c4f26)

**Comment by cungminh2710** [comment]
It's because the secrets aren't available in the fork. Now that you are a contributor, you can create a branch directly in the repo, and run actions on pull request correctly

**Reply by ynguyen2k2** [reply]
Thank you
